modules/rgbd/src/hash_tsdf.hpp

```diff
@@ -60,11 +61,30 @@ struct tsdf_hash
     }
 };
 
+
+//typedef std::unordered_map<cv::Vec3i, VolumeUnit, tsdf_hash> VolumeUnitMap;
+
+typedef unsigned int volumeIndex;
+struct VolumeUnit
+{
+    cv::Vec3i coord;
+    volumeIndex index;
+    cv::Matx44f pose;
+    Vec4i volDims;
```

Comment:
Since all volume units have the same size, we shouldn't keep `volDims` for each of them.
Another question is should we keep `pose` for each of them since it can be calculated from `coord` and volume pose - I think, we shouldn't.

Reply:
volDims - done